frontend-web/webclient/app/Applications/Studio/Applications.tsx

```diff
@@ -124,126 +193,119 @@ export const App: React.FunctionComponent = () => {
 
     const refresh = useCallback(() => {
         setAppParameters(UCloud.compute.apps.findByName({appName: name, itemsPerPage: 50, page: 0}));
-        fetchAllTags(UCloud.compute.apps.listTags({}));
+        setGroups(listGroups({}));
     }, [name]);
 
     useRefreshFunction(refresh);
     useLoading(commandLoading || apps.loading);
 
     const appTitle = apps.data.items.length > 0 ? apps.data.items[0].metadata.title : name;
-    const tags = apps.data.items[0]?.tags ?? [];
+    const flavorName = apps.data.items.length > 0 ? apps.data.items[0].metadata.flavorName ?? appTitle : undefined;
```

Comment:
This is confusing to read. I think some parentheses would improve readability.
I assume it's something like this
```suggestion
    const flavorName = apps.data.items.length > 0 ? (apps.data.items[0].metadata.flavorName ?? appTitle) : undefined;
```